### PR TITLE
Fix broken link for UI initial load table, add more logs in backend

### PR DESCRIPTION
**Pull request by Amogh-Bharadwaj**
Fix link in snapshot status table
Account for mirrorId being a clone job name instead of a regular name in the mirrors page.
Add logs for mirror status and flow state change endpoints
Functionally tested with initial load and qrep

Also improved UI a little bit on qrep page

### CODE REVIEW

**Review by serprex — APPROVED** [review]

File: ui/app/mirrors/[mirrorId]/page.tsx [hunk 1]
```diff
@@ -18,13 +18,20 @@ type EditMirrorProps = {
   params: { mirrorId: string };
 };
 
-function getMirrorStatusUrl(mirrorId: string) {
+const obtainMirrorName = (jobName: string) => {
+  if(jobName.startsWith("clone_")){
+    return jobName.split("_")[1];
```
Comment (heavycrystal) [inline ui/app/mirrors/[mirrorId]/page.tsx:23]: mirrors can have underscores in them, this fails then

Comment (serprex) [reply ui/app/mirrors/[mirrorId]/page.tsx:23]: jobName.slice("clone_".length)

**Comment by Amogh-Bharadwaj** [comment]
functionally tested with initial load and qrep

File: flow/cmd/mirror_status.go [hunk 2]
```diff
@@ -184,11 +194,11 @@ func (h *FlowRequestHandler) cloneTableSummary(
 	var numRowsSynced pgtype.Int8
 	var avgTimePerPartitionMs pgtype.Float8
 
-	rows, err := h.pool.Query(ctx, q, "clone_"+flowJobName+"_%")
+	rows, err := h.pool.Query(ctx, q, "clone_"+mirrorName+"_%")
```
Comment (serprex) [inline flow/cmd/mirror_status.go:197]:
```suggestion
	rows, err := h.pool.Query(ctx, q, "clone_"+mirrorName+"\\_%")
```

**Review by serprex — APPROVED** [review]

**Pull request merged**